src/preamble.js

```diff
@@ -19,6 +19,13 @@ Module.realPrint = out;
 out = err = function(){};
 #endif
 
+#if RELOCATABLE
+{{{ makeModuleReceiveWithVar('dynamicLibraries', undefined, '[]', true) }}}
+#if RUNTIME_LINKED_LIBS
+dynamicLibraries = dynamicLibraries.concat({{{ JSON.stringify(RUNTIME_LINKED_LIBS) }}});
```

Comment:
this flips the order from what it was before, but I guess that should not matter?

i agree with the code like this the order is natural this way. so lgtm.

Reply:
I didn't mean to flip the order..  will fix.